packages/docs/community/release-process/smart-contracts.md

```diff
@@ -37,57 +37,71 @@ When a release is getting proposed on a Baklava/Alfajores/Mainnet, the commit th
 After a completed release process, the release branch should be merged into `master` with a merge commit (instead of the usual squash merge strategy).
 
 
-## Build and Release Process
+## Release Process
 
-Using the following scripts, new contracts can be built and deployed alongside a corresponding on-chain governance proposal. Run the following scripts from the branch with the new release contracts:
+Using the following procedure, a new contract release can be built, deployed, and proposed for upgrade automatically.
 
+### Verify Current Deployment
+
+Compile the contracts in <code>$RELEASE_BRANCH</code> branch and verify that the deployed bytecode on <code>$NETWORK</code> matches compiled bytecode.
 ```bash
-## Script 1 ##
 # TODO: command to pull source code from release N-1, compile and get the artifacts
-# verify the current contract bytecode on the network matches the contracts currently on the RC1 branch
-yarn truffle exec --network $NETWORK ./scripts/truffle/verify-bytecode.js --build_artifacts build/rc1 --before_release_1
+yarn verify-deployed -n $NETWORK -b $RELEASE_BRANCH
+```
 
-## Script 2 ##
-# TODO: command to pull source code from current release branch
-# generate report on backwards compatibility between RC1 and current branch with new release contracts
-yarn ts-node scripts/check-backward.ts report --exclude ".*Test|Mock.*|I[A-Z].*|.*Proxy|LinkedList|SortedLinkedList|SortedLinkedListWithMedian|MultiSig.*|ReleaseGold" -o build/rc1 -n build/contracts --output_file report.json
+### Check Backward Compatibility
+
+Compile the contracts in <code>$RELEASE_BRANCH</code> and <code>$BRANCH</code> branches and check backwards compatibility against a release, subject to the following exceptions:
+  1. If the STORAGE version has changed, do not perform backwards compatibility checks
+  2. If the MAJOR version has changed, check storage layout is backwards compatible, but do not check contract ABI is backwards compatible
 
-## Script 3 ##
-# deploy new contracts and generate governance proposal for upgrade
-yarn truffle exec --network development ./scripts/truffle/make-release.js --build_directory build/ --report report.json --proposal proposal.json --initialize_data example-initialize-data.json'
+For changed contracts, confirm that version number in <code>$BRANCH</code> is strictly greater than <code>$RELEASE_BRANCH</code> version number.
```

Comment:
I'm not too familiar with this step, but wouldn't the script just exit 1 if something is incorrect? I.e. does someone have to manually check something here and is this more meant as explanation/context?

Reply:
no one has to manually check everything
this is explaining what the command does but can be reworded

Reply:
Ah gotcha, maybe its just me but I read it as an instruction to the reader (most things in the doc are IMO)